### ISSUE TITLE
No single pointer is provided to drag the "Copilot github Chat Separator" to the required position: A11y_Visual Studio Code Client_Copilot_Copilot Chat_Dragging movements. 

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode#221558]
### GIT Hub Tags:
#A11yWCAG2.2; #A11yTCS; #A11ySev3; #Visual Studio Code Client; #BM_Visual Studio Code Client_MAC_JULY2024; #DesktopApp; #WCAG2.5.7; #FTP; #A11yMAS; #Dragging Movements;

### Environment and OS details:
MAC Book Pro
MAC OS Sonoma Version 14.5
Visual Studio code Insider version 1.92.0

### Reproduction Steps:

1. Open Visual studio insider
2. In the Visual Studio Code Marketplace, go to the GitHub Copilot extension page and click Install.
3. After installing the extension go to the github copilot chat option available in the left side panel and press enter.
4. Observe that No single pointer is provided to drag the "Copilot github Chat Separator" to the required position.

### Actual behavior:
No single pointer is provided to drag the "Copilot github Chat Separator" to the required position.

### Expected behavior:
There should be single pointer is provided to drag the "Copilot github Chat Separator" to the required position.

### User Impact:
People with motor impairments who cannot operate content because the only way to actuate a function is by dragging a target element from its initial position to some other position will face difficulty in accessing the application.

### Attachments:

https://github.com/user-attachments/assets/73f458c9-f038-4e2c-ac78-910e9f228f8c

**Comment by PoojaNamde** [comment]
#Rev:poojanamde;

**Comment by roblourens** [comment]
Sorry, I don't understand the problem. Can you give an example of what you are looking for? I think this behavior is typical for resizing panes in an application. 

Note that there are also the commands "Increase/Decrease current view size" which can be bound to any keybinding, to resize without dragging.
